src/systems/ec/implementations/vanilla/packages/ec-component-timeline/ec-component-timeline.scss

```diff
@@ -0,0 +1,96 @@
+/**
+ * Timeline
+ * @define timeline
+ */
+
+// Import base
+@import '@ecl/ec-base/ec-base';
+
+@mixin ecl-timeline(
+  $border-width: 0.25rem,
+  $border-color: $ecl-color-yellow-100,
+  $bullet-fill-color: $ecl-color-white-100,
+  $bullet-border-color: $ecl-color-grey-100,
+  $bullet-border-width: 0.25rem,
+  $label-color: $ecl-color-grey-100,
+  $content-color: $ecl-color-grey-100
+) {
+  .ecl-timeline {
+    // Leave space for the bullets
+    margin: 0 0 0 (($ecl-spacing-s / 2) + $bullet-border-width);
+  }
+
+  .ecl-timeline__list {
+    border-left: $border-width solid $border-color;
+    list-style: none;
+    margin: 0;
+    padding-left: 0;
```

Comment:
(if the button is part of the list) From what I see, there's a M vertical padding on the list and then margin-top on list items following other list items. This is I'd implement the spacing here